NOTICE.txt

```diff
@@ -491,6 +318,19 @@ SOFTWARE.
 
 ---
 
+## redux-persist-node-storage
+
+This product contains 'redux-persist-node-storage' by Pelle Jacobs.
+
+Storage adaptor to use node-localstorage with redux-persist
+
+* HOMEPAGE:
+  * https://github.com/pellejacobs/redux-persist-node-storage
+
+* LICENSE: 
```

Comment:
No license

Reply:
This is the one I meant to comment on previously. NPM says MIT for the license, but given there's no license shown in the Github repo I'm not sure what we can assume.

Reply:
I've requested that they add a license file: https://github.com/pellejacobs/redux-persist-node-storage/issues/7.